pkg/config/containers.conf

```diff
@@ -18,7 +18,7 @@
 
 # List of devices. Specified as
 # "<device-on-host>:<device-on-container>:<permissions>", for example:
-# "/dev/sdc:/dev/xvdc:rwm".
+# "/dev/sdc:/dev/xvdc:rwm"..
```

Comment:
Nit: Added an extra period.

Reply:
ish, how'd that get in there ... will fix, i also have a test failure